applicationinsights.json

```diff
@@ -0,0 +1,13 @@
+{
+  "sampling": {
+    "percentage": 100
+  },
+  "instrumentation": {
+    "logging": {
+      "level": "DEBUG"
+    },
+    "micrometer": {
+      "enabled": true
+    }
+  }
+}
```

Comment:
What kind of impact on performance will this have (if any)? i.e. does this need to be turned off for 'production' use, or is it safe to use the same configuration for all instances?

Reply:
They look like parameters we'd want to be able to adjust for sure - are they configurable through environment variables like some of the other app insights params?

Reply:
This can be also set by an Env var like documented here: https://docs.microsoft.com/en-us/azure/azure-monitor/app/java-standalone-config#auto-collected-logging
I can remove it here and push it through the deployment or we can leave that for later?

Reply:
I think this is good for a starter, the controls are available to dial it back if we need to later on.